Documentation/README_v2.md

```diff
@@ -0,0 +1,154 @@
+# etcd
```

Comment:
probably we should just delete this. we will copy the entire v2 doc from 2.3 release to here.